flake.nix

```diff
@@ -0,0 +1,63 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+
+    crane.url = "github:ipetkov/crane";
+    crane.inputs.nixpkgs.follows = "nixpkgs";
+
+    flake-utils.inputs.nixpkgs.follows = "nixpkgs";
+  };
+
+  outputs = {
+    self,
+    nixpkgs,
+    crane,
+    flake-utils,
+  }:
+    flake-utils.lib.eachSystem
+    [
+      flake-utils.lib.system.x86_64-linux
+      flake-utils.lib.system.aarch64-linux
+      flake-utils.lib.system.aarch64-darwin
+    ] (system: let
```

Comment:
Shouldn't this also include x86_64-darwin?

Reply:
thinking about this, i'm not confident about `aarch64-darwin` either as i've not tested it. do you run any CI for darwin?

Reply:
yes it's included in CI, should be x86_64 architecture though I'm not 100% sure of that